### DISCUSSION TITLE
Install as an extension?

### DISCUSSION THREAD

**Discussion by vis223** [AIrjen/OneButtonPrompt#93] [Ideas]
I love this script and use it all the time. 
However, I find it frustrating that I can't use it with other scripts, such as x/y/z grid or infinity grid. 
I know that it's not currently possible to stack scripts, but extensions can be stacked.
Would it be possible to make this script into an extension?

**Comment by AIrjen** [comment]
Hi, yes, if I ever move to a 2.0 version, then I would change it from a script to an extension.
